pdns/README-dnsdist.md

```diff
@@ -971,6 +971,47 @@ latest version of [PowerDNS
 Metronome](https://github.com/ahupowerdns/metronome) comes with attractive
 graphs for `dnsdist` by default.
 
+Query counters
+-------------
+When using `carbonServer`, it is also possible to send per-records statistics of
+the amount of queries by using `setQueryCount(true)`. With query counting enabled,
+`dnsdist` will increase a counter for every unique record or the behaviour you define
+in a custom Lua function by setting `setQueryCountFilter(func)`. This filter can decide
+whether to keep count on a query at all or rewrite the queried record.
+An example of a `QueryCountFilter` would be:
+
+```
+function filter(dq)
+  qname = dq.qname:toString()
+
+  -- don't count PTRs at all
+  if(qname:match('in%-addr.arpa$')) then
+    return false, ""
+  end
+
+  -- rewrite these queries
```

Comment:
Same thing here, it might be useful to clarify that the counting key is rewritten, not the query.